docs/flamingock-library-config/setup-and-stages.md

```diff
@@ -162,6 +162,10 @@ Conditional stage execution based on dependencies or conditions is planned for f
 
 ## Required fields
 
+Each `@EnableFlamingock` annotation must define:
```

Comment:
Remove `each`, sounds like the user can add more than one `@EnableFlamingock` annotation to the application.